benchmark/README.md

```diff
@@ -299,18 +299,44 @@ the runtime of the benchmark. Run `bench.start()` after the initialization
 and `bench.end(n)` when the benchmark is done. `n` is the number of operations
 you performed in the benchmark.
 
+Note: the benchmark script will be run twice. The first pass will configure
+the benchmark and WILL NOT run the `main` function. In this pass,
+no flags except the ones directly passed via commands that you
+run the benchmarks with will be used. In the second pass,
+the `main` function will be run, and the process will be run with:
+
+*  `--expose_internals`
```

Comment:
Yeah, I think it's simpler. (And from what I've grepped, there is only one benchmark uses this :P)